src/sentry/options/defaults.py

```diff
@@ -815,7 +815,9 @@
 register("store.background-grouping-sample-rate", default=0.0, flags=FLAG_AUTOMATOR_MODIFIABLE)
 
 # True if background grouping should run before secondary and primary grouping
-register("store.background-grouping-before", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE)
+register(
+    "store.background-grouping-before", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE
+)  # unused
```

Comment:
nit: use `TODO` because many tools will automatically surface that 
```suggestion
)  # TODO: remove, no longer used
```

Reply:
Good call. I'm about to get the `TODO` to-done, but you're right that it'd be a good habit to get into.